MAINTAINERS.md

```diff
@@ -39,8 +39,8 @@ If you are an owner of the organization, you should be able to see an automated
   [19 November 2019](https://github.com/tldr-pages/tldr/issues/3580) - present
 - **Proscream ([@Proscream](https://github.com/Proscream))**:
   [22 November 2019](https://github.com/tldr-pages/tldr/issues/3592) - present
-- **Zlatan Vasović ([@zdroid](https://github.com/zdroid))**:
-  [30 November 2019](https://github.com/tldr-pages/tldr/issues/3636) - present
+- Zlatan Vasović ([@zdroid](https://github.com/zdroid)):
+  [30 November 2019](https://github.com/tldr-pages/tldr/issues/3636) - [17 December 2019](https://github.com/tldr-pages/tldr/issues/3663)
```

Comment:
```suggestion
```
We usually *remove* people from the "collaborators" list when they become members. Please remove this line.

Reply:
Oh yeah, good point. I always forget these. Nice catch.

Reply:
Thanks for suggestion, I wasn't sure about this one.

Reply:
Wouldn't it be better to keep the line for future record, though? Otherwise it will be unnecessarily hard to track down.

Reply:
By the way, I think last time discussed this, the conclusion was to use bold to denote current members of a group, and plain formatting for the former ones. That's already the case for the organization members and owners section of this document, so it's only consistent to apply the same logic here as well.